src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1947,6 +1947,15 @@ default MessageCreateAction replyFiles(@Nonnull Collection<? extends FileUpload>
      */
     boolean isEphemeral();
 
+    /**
+     * Whether this message is silent.
+     * <br>The message being silent means it will not trigger push and desktop notifications
+     * <br>This is a shortcut method for checking if {@link #getFlags()} contains {@link MessageFlag#NOTIFICATIONS_SUPPRESSED}
+     *
+     * @return Whether the message is silent
+     */
+    boolean isSilent();
```

Comment:
It seems like we have settled on calling this feature `isSuppressedNotifications` to align with `isSuppressedEmbeds`, could you update all the getter and setter names to follow this naming scheme instead?

Reply:
Done.